Telegram/SourceFiles/codegen/common/CMakeLists.txt

```diff
@@ -5,4 +5,5 @@ add_library(codegen_common STATIC
     cpp_file.cpp
     logging.cpp
 )
-qt5_use_modules(codegen_common Core)
+target_link_libraries(codegen_common Qt5::Core)
```

Comment:
You have order reversed here. `find_package` must be  first.

Since it works anyway, just remove `find_package` below and verify that it's included in the higher level, most probably it's in the top-level cmakelists.